src/pages/act-now.js

```diff
@@ -42,6 +43,7 @@ export async function getStaticProps({ locale }) {
 
   const page = await wp().pages({ slug: "act-now", locale }).first;
   const { actNow = {} } = page;
+  const petitions = await actnow().petitions().list;
```

Comment:
Since this page is starting to look like it depends on if and who is logged in, may be we should switch to `getServerSideProps`?